meta/src/parser.rs

```diff
@@ -1093,13 +1132,43 @@ mod tests {
         };
     }
 
+    #[test]
+    fn grammar_doc_and_line_doc() {
```

Comment:
not sure if there could also be an assertion to check the comment was asserted correctly?

Reply:
Sorry, I don't know what your means of this. Would you please explain more with informations?